app/controllers/residential_address_controller.rb

```diff
@@ -21,6 +21,6 @@ def snap_application_param
   end
 
   def skip?
-    current_snap_application.mailing_address_same_as_residential_address?
+    current_snap_application&.mailing_address_same_as_residential_address?
```

Comment:
I'm guessing yes(?), but the tests surfaced this as the only place where this caused an issue.